src/libawkward/builder/ArrayBuilder.cpp

```diff
@@ -1,493 +1,411 @@
-// BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+// BSD 3-Clause License; see
+// https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
 
-#define FILENAME(line) FILENAME_FOR_EXCEPTIONS("src/libawkward/builder/ArrayBuilder.cpp", line)
+#define FILENAME(line)                                                         \
+  FILENAME_FOR_EXCEPTIONS("src/libawkward/builder/ArrayBuilder.cpp", line)
 
 #include <sstream>
 #include <stdexcept>
 
-#include "awkward/common.h"
+#include "awkward/BuilderOptions.h"
+#include "awkward/builder/ArrayBuilder.h"
 #include "awkward/builder/Builder.h"
 #include "awkward/builder/UnknownBuilder.h"
-#include "awkward/builder/ArrayBuilder.h"
-#include "awkward/BuilderOptions.h"
+#include "awkward/common.h"
 
 namespace awkward {
-  ArrayBuilder::ArrayBuilder(const BuilderOptions& options)
-      : builder_(UnknownBuilder::fromempty(options)) { }
+ArrayBuilder::ArrayBuilder(const BuilderOptions &options)
+    : builder_(UnknownBuilder::fromempty(options)) {}
 
-  const std::string
-  ArrayBuilder::to_buffers(BuffersContainer& container, int64_t& form_key_id) const {
-    return builder_.get()->to_buffers(container, form_key_id);
-  }
+const std::string ArrayBuilder::to_buffers(BuffersContainer &container,
+                                           int64_t &form_key_id) const {
+  return builder_.get()->to_buffers(container, form_key_id);
+}
 
-  int64_t
-  ArrayBuilder::length() const {
-    return builder_.get()->length();
-  }
+int64_t ArrayBuilder::length() const { return builder_.get()->length(); }
 
-  void
-  ArrayBuilder::clear() {
-    if (builder_) {
-      builder_.get()->clear();
-    }
+void ArrayBuilder::clear() {
+  if (builder_) {
+    builder_.get()->clear();
   }
```

Comment:
I'm glad that one-line if statements still get the braces. We should use braces all the time, uniformly.

```c++
  if (builder_)
    builder_.get()->clear();
```

is dangerous.